jumeg_math.py

```diff
@@ -81,4 +79,44 @@ def calc_tkeo(signal):
     tkeo[-1] = 0.
 
     # return results
-    return tkeo
\ No newline at end of file
+    return tkeo
+
+
+##################################################
+#
+# destroy phase/time info on any data array
+#
+##################################################
+def shuffle_data (data_trials, mode='shuffle'):
+    ''' Shuffling the time information (phase) of any data array
+    Parameters
+    ----------
+    data_trials : 2d ndarray of dimension [ntrials x nsamples]
+    mode : 2 different modi are allowed.
+    'mode=shuffle' whill randomly shuffle the phase values
+    'mode=shift' whill randomly shift the phase values
+
+    Returns
+    -------
+    s_trial : shuffled (phase) trials
+
+    '''
+
+    np.random.seed()     # for parallel processing it needs to be re-initialized
```

Comment:
Is this method of initialization enough? Maybe we should also provide an option to enter the seed value to reproduce results.